deploy/charts/arcadia/values.yaml

```diff
@@ -80,3 +80,8 @@ portal:
     issuerURL: https://portal.172.22.96.167.nip.io/oidc
     clientSecret: 61324af0-1234-4f61-b110-ef57013267d6
     masterURL: https://k8s.172.22.96.167.nip.io
+
+vectorstore:
```

Comment:
@Abirdcfly  please double check this field

Reply:
Do we need add chroma chart as dependence, like minio?

Reply:
@Abirdcfly we can add this chroma chart in another pr.  https://github.com/amikos-tech/chromadb-chart 
Do we need this charts downloaded and uploaded in charts dir or only a `dependency` works too?